A minor change to the output comments in Example-1 of C# Properties guide

themasterchiefdev opened the pull request:
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

themasterchiefdev committed:
a minor update to the output comments

Changed the output from John Doe --> Wayne Withow

themasterchiefdev committed:
Merge pull request #1 from yvrkarthik/yvrkarthik-patch-1

a minor update to the output comments

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

guardrex reviewed (changes requested):
@yvrkarthik Excellent catch on the output name not matching the code. Can we go with the name "Isabelle Butts" here? ... It's on the MS approved names list, and I :heart: the name ... If "Isabelle Butts" doesn't bring a smile :smile: to the reader, nothing will! lol If you're cool with it, change it here and in the code on line 21.

themasterchiefdev committed:
updated the output,firstname and lastname in Example-1 of C# Properties guide

As per the chanages proposed by @GuardRex , 
I have updated the code on line 21 in the main method from--  new Person("Wayne", "Withrow") --> new Person("Isabelle", "Butts") 
and updated the output comment from  Wayne Withrow --> Isabelle Butts

GuardRex commented:
@yvrkarthik :grinning: I appreciate you going along with that. Now, we're both in hot water :fire: if it doesn't go over well! lol I'm going to merge this really fast 🏃 as soon as the build is done. Thanks for helping out with this!
